File: Exercícios - Slide 2/8.c
// Ler dois números n1 e n2
// Exibir todos os números entre n1 e n2
// Fazer em Python e em C

#include <stdio.h>

int main() {
    int n1, n2;
    printf("Digite um numero\n");
    scanf("%d", &n1);
    printf("Digite outro numero\n");
    scanf("%d", &n2);
    for (int i = n1+1; i<n2; i++) {
        printf("%d ", i);
    }
}

File: Exercícios - Slide 2/8.py
# Ler dois números n1 e n2
# Exibir todos os números entre n1 e n2
# Fazer em Python e em C

n1 = input("Digite um numero\n")
n2 = input("Digite outro numero\n")

for i in range(int(n1)+1, int(n2)):
    print(i, end=" ")